spec/spec_helper.rb

```diff
@@ -1,7 +1,19 @@
 require "simplecov"
 require "pry"
 SimpleCov.start do
-  track_files "{app,lib}/**/*.rb"
+  add_filter '/db/'
```

Comment:
We do actually want to assert coverage on `db/seeds` so please don't filter out all of `db`

Reply:
I removed the "/db" filter.